src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackSessionsPage.java

```diff
@@ -157,7 +157,8 @@ public void addFeedbackSession(FeedbackSessionAttributes newSession, boolean isU
         waitForElementPresence(By.cssSelector("#instructions iframe"));
 
         if (isUsingTemplate) {
-            selectDropdownOptionByText(sessionTypeDropdown, "session using template: team peer evaluation");
+            selectDropdownOptionByText(sessionTypeDropdown,
+                    "session using template: team peer evaluation (percentage-based)");
         } else {
             selectDropdownOptionByText(sessionTypeDropdown, "session with my own questions");
         }
```

Comment:
Best to add an e2e for the new template as well.

Reply:
Full path coverage is not the aim of E2E test; in fact, that's one of the anti-patterns of E2E test. Is there anything specifically from the new template that has to be verified at E2E level?

Reply:
Noted. Nothing in particular, just seems off that the default template doesn't have an e2e test but the legacy one does. Let's at least update the template here to the new default one?

It would be good to add a test somewhere though, whether the template questions load correctly should be tested at some level. Can we add a frontend test for this in this case?

Reply:
I see what you mean. Sorry I should have taken a look at the relevant code first. 

Frontend tests don't necessarily mean snapshot tests, we can also check that the relevant methods are called with the correct arguments as well.

However, looking at the code, I also can't think of any clean way to test whether templated feedback sessions are created correctly without also creating tests for checking whether `addNewSessionHandler` works in general, which seems a little out of scope for this PR. Also, seems like there are currently no tests for that particular page.

I think it's best we left this to #10920 where both these issues can be handled at once. @wkurniawan07 any advice?